### ISSUE TITLE
Composition: 'none' missing from the documentation for waapi.animate

### ISSUE THREAD

**Issue by 97Yates05** [juliangarnier/anime#751]
What I mean by that is that an element cannot return to its original position after the animation is over.The alternate option also animates once on return.
For example, after moving 50px to the left, I need the element to return to its original position immediately, which I can only do manually in the Complete callback.

**Comment by cameralibre** [comment]
@97Yates05 There isn't enough information in this issue for others to provide you much help. 

Please fill in the prompts in the body text of your issue to make it easier, (eg: _A clear and concise description of what the problem is._)

Anime.js almost certainly _can_ do whatever it is you're trying to do. You can read the anime.js documentation for help (it sounds like you want to set `direction: alternate`) 

Documentation: [Animation Parameters](https://animejs.com/documentation/#direction)

**Comment by 97Yates05** [comment]
@cameralibre The problem has been updated, please pay attention.The sample code is as follows
```
anime({
  target: '#element', 
  translateY:50px,
  duration:1500,
  complete:()=>{
    anime.set({
       translateY:0,
    });
  }
})
```

**Comment by cameralibre** [comment]
That information is a lot more useful, thanks.

An alternative could be to use keyframes rather than using a callback: 
```javascript
anime({
  targets: '#element', 
  keyframes: [
      { translateY: '50px',  duration: 1500 },
      { translateY: '0',  duration: 0 }
  ]
})
```

see also [codepen version](https://codepen.io/cameralibre/pen/ZEBGLZw?editors=1010)

But it sounds like you are making a feature request for anime to provide an option like CSS' [`animation-fill-mode`](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-fill-mode) - is that right?

**Comment by 97Yates05** [comment]
Yes, I expect the animation-fill-mode property to be none.Animejs, however, are all forward by default.

**Comment by cameralibre** [comment]
So there are two things you're suggesting:
1. Provide an `animationFillMode` option
2. Make `animationFillMode: none` the default in anime.js

The first seems feasible, though you'll need to explain more clearly in what contexts setting an `animationFillMode` option is preferable or necessary, rather than using the 'manual' approaches we listed above.

As for the second suggestion, (making `none` the default), it might be difficult to convince people - personally, I like `animation-fill-mode` being set to `forwards`, as I predominantly use anime.js for timelines. (But I don't develop anime, so I'm not someone you need to convince 🙂️) Again, you'll need to provide context and clear examples of why it should be that way. 

In order to make it clearer to people developing anime.js, I would also suggest editing the title of this issue to something like **Default to 'animation-fill-mode: none'**. The current title, '**Why don't element go back to the original place after animation? Isn't that normal behavior**' makes the issue sound like a usage problem rather than a feature request.

**Comment by juliangarnier** [comment]
Sorry for the wait, v4 introduced a `composition` parameter, but I haven't added a fill mode that does what you're asking in the JavaScript version of the animate method.
But, you can use the native `wappi.animate()` version that does accepts native `fill` values, like 'none'.

I'm realizing that I haven't documented this properly, but the following should be working:

```js
waapi.animate(target, {
  x: 100,
  composition: 'none'
});
```

v4 also introduces a utility function that removes the inline styles, you can do something like this:

```js
animate(target, {
  x: 100,
  onComplete: utils.cleanInlineStyles
});
```

Check out the new [documentation](https://animejs.com/documentation/) and the [migration guide from v3](https://github.com/juliangarnier/anime/wiki/Migrating-from-v3-to-v4).
